package.json

```diff
@@ -35,7 +37,8 @@
     "@types/node": "^14.0.0",
     "jest": "^23.1.0",
     "ts-jest": "^26.0.0",
-    "typescript": "^2.9.2"
+    "typescript": "^2.9.2",
+    "nexe": "^3.3.7"
```

Comment:
Would you describe where we use this `nexe`?

Reply:
I used it to generate a binary locally to test some things, it doesn't need to be included so I just removed it.